inbm-lib/inbm_common_lib/utility.py

```diff
@@ -124,7 +124,7 @@ def remove_file(path: Union[str, Path]) -> bool:
         return False
 
 
-def create_file_with_contents(path: Union[str, Path], contents: List[str]) -> None:
+def create_file_with_contents(path: Union[str, Path], contents: list[str]) -> None:
     """ Create a file and add the contents by line
```

Comment:
**Suggestion:** Use a context manager when opening a file to ensure it is properly closed in case of an exception. [best practice]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
with open(canonical_path, 'w') as f:
    for string in contents:
        f.write(f"{string}\n")
```